Documentation/Configuration.md

```diff
@@ -82,6 +82,10 @@ An example `.swift-format` configuration file is shown below.
     "blankLineBetweenMembers": {
         "ignoreSingleLineProperties": true
     },
+    "blankLineAfterFunctionDeclarations": {
```

Comment:
Technically, the "function declaration" includes everything from the first token (arguably even the preceding doc comment) up through the body and the closing brace. So I was a bit confused when I first read this description, because to me, "after function declaration" would imply "after the final `}`", rather than "after the opening `{` that follows the function signature".

A clearer name for this might be `blankLineAtBeginningOfFunctionBody`.

Reply:
I agree. Still getting used to all this new syntax _lingo_ 😅